modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -146,8 +146,8 @@ def id_str(self):
     def get_dtype(self, col):
         # If we search for an index column type in a MultiIndex then we need to
         # extend index column names to tuples.
-        if isinstance(self._dtypes, MultiIndex) and not isinstance(col, tuple):
-            return self._dtypes[(col, *([""] * (self._dtypes.nlevels - 1)))]
+        if isinstance(self._dtypes.index, MultiIndex) and not isinstance(col, tuple):
```

Comment:
fixing a typo here: `self._dtypes` is supposed to be a Series mapping column names to its dtypes